RetailCoder.VBE/Extension.cs

```diff
@@ -70,7 +70,6 @@ public void OnStartupComplete(ref Array custom)
         public void OnDisconnection(ext_DisconnectMode RemoveMode, ref Array custom)
         {
             _app.Shutdown();
-            _app.Dispose();
             _kernel.Dispose();
```

Comment:
I wonder if `OnDisconnection` is consistently/reliably called in all situations.

I suppose this runs normally when the host app is X'd-out. But does it run when the debugger is abruptly stopped (i.e. the process is killed)? Does it even matter? hmm..